lib/colors.ts

```diff
@@ -56,6 +56,8 @@ const colorMap: Record<string, string> = {
   blueviolet: '#8A2BE2',
   brown: '#A52A2A',
   burlywood: '#DEB887',
+  burgandy: '#800020',
+  burgundy: '#800020',
```

Comment:
Duplicated? Or 👍 ?

Reply:
That's the way i found the color values are translated too (by the plugins if you choose an automatic one), so i put the translated one to test and forgot to take `burgandy` out. Used that map to convert string to hex, and check if the color is valid. Then if not to show the text, like:  https://commerce-shopify-334ih0w49-vercel-solutions-vtest314.vercel.app/product/macbook-pro later on more colors can be aded, and also based on locale to have translated versions. Should i make a commit to take that out ?